### PR TITLE
Reader: Add byline to Feed Header

### PR DESCRIPTION
**Pull request by jancavan**
The new Feed Header needs to have a byline added to it:

screenshot 2016-11-22 12 01 46

- Display Author's First Last name
- If there isn't one, fall back to username
- If there are multiple Authors, display primary owner

### CODE REVIEW

**Comment by blowery** [comment]
> If there are multiple Authors, display primary owner

We don't expose that info via the API currently. And for feeds from purely external sites, we don't even have it. What should we do?

**Comment by fraying** [comment]
@blowery Okay to skip it if we don't have it.

**Comment by bluefuton** [comment]
We don't currently get any author information from the API about a feed or site. 

We could potentially find the first post available and strip the author information from that. Unfortunately we don't store posts in a format that allows this (`FeedPostStore` needs a post ID).

We could strip the author from the first post in the feed API endpoint, and return the author information there. Might be a performance hit.

**Comment by bluefuton** [comment]
I've added owner display name to the site endpoint (Automatticians: you'll need to apply patch D3498 to see this). It'll only be available for wpcom sites.

It looks like this:

screen shot 2016-11-25 at 15 53 11

**Comment by samouri** [comment]
@bluefuton - I don't see any logic to fallback to username.  Do we still plan on supporting that?

**Comment by blowery** [comment]
@samouri `display_name` should handle that for us in the API.

File: client/blocks/reader-feed-header/index.jsx [hunk 1]
```diff
@@ -61,7 +61,8 @@ class FeedHeader extends Component {
 			feed = this.props.feed,
 			headerImage = site && site.getIn( [ 'options', 'header_image' ] ),
 			headerColor = site && site.getIn( [ 'options', 'background_color' ] ),
-			followerCount = this.getFollowerCount( feed, site );
+			followerCount = this.getFollowerCount( feed, site ),
+			ownerDisplayName = site && site.get( 'owner_display_name' );
```
Comment (blowery) [inline client/blocks/reader-feed-header/index.jsx]: needs updating now with the patch

**Comment by blowery** [comment]
assuming this gets updated to use the full author object, 🚢

**Pull request merged**